### PR TITLE
Disassembler wrapper for optimization purposes

### PR DESCRIPTION
**Pull request by cahirwpz**

### CODE REVIEW

File: tools/disass.py [hunk 1]
```diff
@@ -0,0 +1,795 @@
...
+from bisect import bisect
+from collections import namedtuple, defaultdict, UserString, UserList
+
+from pygments import lexers
+from pygments import formatters
+
+from pyparsing import (Word, Char, Literal, Opt, ZeroOrMore, OneOrMore, Regex,
+                       Suppress, Combine, White, Group)
+from pyparsing.exceptions import ParseException
+from blessings import Terminal
```
Comment (Kristopher38) [inline tools/disass.py:16]: `blessing` isn't installed by default. It'd be a good idea to perhaps add this to `requirements.txt` or some kind of `requirements-dev.txt`

File: effects/README.md [hunk 2]
```diff
@@ -24,3 +22,305 @@ Amiga 500 knowledge base
...
 3. Subpixel line drawing using Blitter: [part 1](http://scalibq.wordpress.com/2011/12/28/just-keeping-it-real-part-5/) and [part 2](http://scalibq.wordpress.com/2012/01/06/just-keeping-it-real-part-5-1/).
 
+### MC68000 instruction timing
+
+Taken from [EAB post](https://eab.abime.net/showthread.php?t=103006).
+
+```
++---------------------+----+----+-----+-----+-----+------+--------+-----+-----+------+--------+----+----+----+----+
+|mnemonic |syntax     |dy  |ay  |(ay) |(ay)+|-(ay)|(d,ay)|(d,ay,r)|(m).w|(m).l|(d,pc)|(d,pc,r)|#d  |ccr |sr  |usp |
++---------+-----------+----+----+-----+-----+-----+------+--------+-----+-----+------+--------+----+----+----+----+
```
Comment (Kristopher38) [inline effects/README.md:32]: The whole table could be rather easily converted to markdown for a nicer display

**Review by Kristopher38 — COMMENTED** [review]
Excellent tool! I like everything about it. There's one more thing I'd like to see - number of cycles for a given function, just a sum of cycles of all instructions, regardless of whether there are e.g. mutually exclusive code paths or loops, just to have a general overview.

File: effects/README.md (hunk 2 above)
Comment (cahirwpz) [reply effects/README.md:32]: Good idea, but let's move it to `demoscene.wiki` first.

File: tools/disass.py (hunk 1 above)
Comment (cahirwpz) [reply tools/disass.py:16]: Done!

**Pull request merged**